noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/event.nr

```diff
@@ -297,3 +291,92 @@ where
         context.emit_private_log(encrypted_log);
     }
 }
+
+// These utils got copied here because note encryption diverged from the event one. The encryption functionality
```

Comment:
Copied these over here as they diverged from the note ones due to the shrunk header (no longer contains contract address).